contrib/flo-styx/pom.xml

```diff
@@ -35,5 +39,13 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
```

Comment:
I feel this make flo-styx quite opinionated/biased towards dataflow. Is it really necessary?

Reply:
There aren't many things in styx that are about data really. It works great for data but can be used for whatever you like. The same applies even more to flo. There is nothing concerning data in flo. Having separate modules that add functionality is fine but steering it too much into a specific use-case (data, styx, dataflow …) will not be good in the long run I think.